src/components/BaseComponents/ReadOnlyValue/ReadOnlyValue.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+interface ReadOnlyValueProps {
+  label: string;
+  value: string;
+}
+
+export default function ReadOnlyValue(props: ReadOnlyValueProps) {
+  const { label, value } = props;
+
+  return (
+    <div className='govuk-summary-list'>
+      <div className='govuk-summary-list__row'>
+        <dt className='govuk-summary-list__key'>{label}</dt>
+        <dd className='govuk-summary-list__value'>{value}</dd>
+      </div>
+    </div>
+  );
+}
```

Comment:
why you are not using ReadOnlyDisplay.tsx component. I can see both are doing same thing.

Reply:
cause this will be customised in the future to do a different task..  it was a decision that was made for future proof :)